ProgramParsing/CONSTANTS/Constants.py

```diff
@@ -0,0 +1,9 @@
+"""
```

Comment:
Set parse_year_beg to 2019 and end to 2021 (we parsing the last 3 years as def right from 2019-2020, 2020-2021, to 2021-2022